requirements.txt

```diff
@@ -1,10 +1,13 @@
 Django==2.1.2
-djangorestframework==3.8.2
-JPype1==0.6.3
-pytz==2018.5
+jpype1==0.6.3
+numpy==1.16.4
```

Comment:
Why numpy?

Reply:
@rtgdk it is in the requirments.txt of the python2 version. Do we not need it?

Reply:
My bad, was looking at the displayed diff, should have checked diff b/w master and this branch.